requirements.txt

```diff
@@ -1,50 +1,58 @@
+#
+# This file is autogenerated by pip-compile with python 3.8
+# To update, run:
+#
+#    pip-compile --allow-unsafe --no-annotate
+#
+async-timeout==4.0.2
+attrs==21.4.0
+backports-zoneinfo==0.2.1
 blinker==1.4
-black==22.3.0
 certifi==2022.6.15
-chardet==5.0.0
+cffi==1.15.1
+charset-normalizer==2.1.0
 click==8.1.3
 clickhouse-driver==0.2.4
-colorama==0.4.5
 confluent-kafka==1.7.0
-coverage==6.4.1
 datadog==0.44.0
-deprecation==2.1.0
-docopt==0.6.2
-flake8==4.0.1
-Flask==2.1.2
-future==0.18.2
+deprecated==1.2.13
+flask==2.1.2
+freezegun==1.2.1
 honcho==1.1.0
 idna==3.3
-isort==5.10.1
+importlib-metadata==4.12.0
+importlib-resources==5.9.0
 itsdangerous==2.1.2
-Jinja2==3.1.2
+jinja2==3.1.2
 jsonschema==4.7.1
-lz4==4.0.1
-Markdown==3.3.7
-MarkupSafe==2.1.1
+markdown==3.3.7
+markupsafe==2.1.1
+milksnake==0.1.5
```

Comment:
I noticed there's a bunch of packages like this that weren't in the requirements before. Is the plan to somehow filter these out?

Reply:
these are the transitive dependencies which on master are currently unpinned

Reply:
Got it.